tests/test-sessions/test-infinispan-sessions/src/test/java/org/eclipse/jetty/server/session/InfinispanTestSupport.java

```diff
@@ -113,7 +113,7 @@ public void setup() throws Exception
                 .persistence()
                 .addSingleFileStore()
                 .location(_tmpdir.getAbsolutePath())
-                .storeAsBinary()
+                .storeAsBinary().enable()
```

Comment:
Can we have a test both with and without storeAsBinary enabled, to be on the safe side?

Reply:
Yes. It just triggers serialization, so data in the cache is in serialized form

Reply:
I mean can you make a test both with and without storeAsBinaryEnabled please.